package.json

```diff
@@ -42,9 +43,11 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hook-form": "7.39.5",
-    "react-select": "5.6.1"
+    "react-select": "5.6.1",
+    "webpack": "5.9.0"
```

Comment:
why is webpack installed? We have replaced it with SWC

Reply:
It was an unmet peer dependency, it was producing a warning.